qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
ok, I understand now, pt-pt really don't existe as a code in any of those systems. I'm just contributing to the 'pt' :)
pt by netolazaro, Pedro Mendonça
I just didn't want to be credited for the pt-br, as I didn't do anything on it, and it sure needs someone to pick it up and fills it 100%.

I was assuming that 'netlazaro' was the one who did create the pre existent pt-br translation, but I couldn't find his refference anywhere to correctly credit.